plugins/filters/add_prefix.py

```diff
@@ -0,0 +1,17 @@
+from ansible import utils
+
+
+def add_prefix(value, prefix=''):
+    """Add string prefix to item.
+
+    {{ ['hanukkah', 'easter', 'pessah'] | map('add_prefix', 'happy ' }}
+    -> ['happy hanukkah', 'happy easter', 'happy pessah']
+    """
+
+    return prefix + value
```

Comment:
If an object defines __str__ but it's not type of String you will get error using +, concentrating strings using above will properly do so. Just one example. Performance wise is not relevant here as it's not too many concentrated strings, but just in general it's + is O(n^2) and above is O(n).